src/components/data/data-utils.js

```diff
@@ -375,6 +377,30 @@ export const EntityUploader = ({ onSuccess, onDismiss, namespace, name, entityTy
           `Data with the type '${newEntityType}' already exists in this workspace. `,
           'Uploading more data for the same type may overwrite some entries.'
         ]),
+        currentFile && containsNullValues && _.includes(_.toLower(newEntityType), entityTypes) && div({
+          style: { ...warningBoxStyle, margin: '1rem 0 0.5rem' }
+        }, [
+          icon('warning-standard', { size: 19, style: { color: colors.warning(), flex: 'none', marginRight: '0.5rem', marginLeft: '-0.5rem' } }),
+          'We have detected empty cells in your TSV. Please choose an option:',
+          div({ style: { paddingTop: '0.5rem' } }, [
+            h(RadioButton, {
```

Comment:
Since these two radio buttons are mutually exclusive options, they should have the same `name` and different `value`s. Using the same name for both will make them part of a group and allow using the arrow keys to switch between options when the group is focused and will cause a screen reader to say the proper number of options.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/radio#defining_a_radio_group

Reply:
I think that you either want to put these in a fieldset, or you need to declare a role of group per https://stackoverflow.com/questions/48265742/how-to-group-form-inputs-accessibly-without-a-fieldset.

Reply:
To clarify, using the same `name` is necessary for the screen reader to properly state the number of options. But you still need to link the legend/title to the choices by either using a fieldset with legend, or by using a group with `aria-labelledby` or `aria-label`.

Reply:
Thanks all, this should be resolved in 17827ccdb918bfcda60d80c3d9d885de878ee386

Tested by tabbing through the page, and the up/down arrow keys allowed me to cycle through the radio options